Dockerfile

```diff
@@ -3,6 +3,20 @@ FROM bitwalker/alpine-elixir:1.4.2
 MAINTAINER PRX <sysadmin@prx.org>
 LABEL org.prx.app="yes"
 
+# install git, aws-cli
+RUN apk update && apk --update add \
+    ca-certificates \
+    git
+
+RUN apk --no-cache update && \
+    apk --no-cache add python py-pip py-setuptools ca-certificates groff less && \
+    pip --no-cache-dir install awscli && \
+    rm -rf /var/cache/apk/*
+
+# install PRX aws-secrets scripts
+RUN git clone -o github https://github.com/PRX/aws-secrets
```

Comment:
Agreed. I probably only need 1 of the scripts in the project to pull and set the env, not all of the files under bin should be copied to the image. So I will not be using this method forever, was just getting them all there to start and for messing around.